docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
+
+Rationale
+---------
+
+There is often the case where a set of documentation has been built,
+and it is kept online but grows outdated over time.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
```

Comment:
Should "old official docs that are unmaintained" be kept if they're the only docs that exist and if the software project is just as old and unmaintained?

Reply:
If I understand correctly, the official and _unofficial_ documentation will be keep. However, the _unofficial_ will be de-listed via `robots.txt`. I don't think the policy mentions deleting the unofficial docs.

Reply:
I think that we also now have a definition of unmaintained that makes more sense - it's not that no one has updated the documentation for a long time... it's that the actual software project has been updated beyond what is reflected in the documentation.